client/home/lib/myteam/components/hometeamsendinvites/view.coffee

```diff
@@ -25,7 +32,7 @@ module.exports = class HomeTeamSendInvitesView extends React.Component
     userEmailClassName = 'kdinput text user-email'
 
     <div className='kdview invite-inputs'>
-      <input type='text' className={userEmailClassName} placeholder='mail@example.com' value={inviteInput.get 'email'} onChange={@props.onInputChange.bind(this, rowIndex, 'email')} />
+      <input type='text' className={userEmailClassName} placeholder='mail@example.com' value={inviteInput.get 'email'} onChange={@onInputChange.bind(this, rowIndex, 'email')} />
```

Comment:
There is nothing wrong with this line, but the handler might need changes. I added my comments over there.